modules/features/account/src/main/java/au/com/shiftyjelly/pocketcasts/account/onboarding/components/SubscriptionTierPill.kt

```diff
@@ -69,7 +69,8 @@ fun SubscriptionTierPill(
 @Composable
 private fun SubscriptionTierPillPreview() {
     SubscriptionTierPill(
-        iconRes = IR.drawable.ic_patron,
-        shortNameRes = LR.string.pocket_casts_patron_short,
+        iconRes = R.drawable.ic_patron,
+        shortNameRes = au.com.shiftyjelly.pocketcasts.localization.R.string.pocket_casts_patron_short,
```

Comment:
Might want to use `LR.string` here

Reply:
Fixed in 8ed61e9

In addition, I renamed the class to `SubscriptionBadge` and displayed the badge only for Patron on the profile header.